script/build.js

```diff
@@ -37,7 +37,7 @@ const smith = Metalsmith(__dirname); // eslint-disable-line new-cap
 
 const optionDefinitions = [
   { name: 'buildtype', type: String, defaultValue: 'development' },
-  { name: 'mergedbuild', type: Boolean, defaultValue: false },
+  { name: 'brand-consolidation-enabled', type: Boolean, defaultValue: false },
```

Comment:
I change the command variable to `brand-consolidation-enabled` for consistency with the naming conventions throughout the site, and hyphenated it to keep it explicit as the top-level command arg.